rdr_service/dao/retention_status_import_failures_dao.py

```diff
@@ -0,0 +1,19 @@
+from rdr_service.dao.base_dao import BaseDao
+from rdr_service.model.retention_status_import_failures import RetentionStatusImportFailures
+
+
+class RetentionStatusImportFailuresDao(BaseDao):
+
+    def __init__(self):
+        super(RetentionStatusImportFailuresDao, self).__init__(RetentionStatusImportFailures)
+
+    def insert(self, obj: RetentionStatusImportFailures):
+        with self.session() as session:
+            self.insert_with_session(session, obj)
+
+    def get(self, failure_id):
```

Comment:
Can we rename this to signify what the method is doing... ex `get_obj_from_failure_id`